src/ngraph_mark_for_clustering.cc

```diff
@@ -204,10 +204,25 @@ Status MarkForClustering(Graph* graph,
   // needed in case the graph is broken up in a later rewrite pass (for example,
   // constant data).
 
+  static std::set<string> disabled_ops_set = {};
+
+  auto disabled_ops_list =
+      ng::split(string(config::ngraph_get_disabled_ops()), ',');
+  // In case string is '', then splitting yields ['']. So taking care that ['']
+  // corresponds to empty set {}
+  set<string> disabled_ops_set_current = {};
+  if (disabled_ops_list.size() >= 1 && disabled_ops_list[0] != "") {
+    disabled_ops_set_current =
+        set<string>(disabled_ops_list.begin(), disabled_ops_list.end());
+  }
```

Comment:
Can this clean up be moved to the get or set utility?  So that we can directly get the set of disabled op types.